storybook/src/docs/borders.docs.mdx

```diff
@@ -2,9 +2,9 @@
 
 import { Meta } from "@storybook/blocks";
 
-<Meta title="Docs/Design Guidelines/Borders" />
+<Meta title="Foundation/Design Tokens/Border" />
```

Comment:
What you're describing here are not necessarily the design tokens, but the design I think. Design tokens are a means to implement the design.

Reply:
Not just a means to and end – they are the essential individual choices within ‘the design’.

‘Design token’ has become a well-known term in the DS world. It is a major and public building block of the NLDS architecture.

A soon as we add those fancy previews of colours, spacing widths etc. I prototyped in that documentation model branch, the actual tokens will be more visible on these pages as well and they will do the title justice.